src/extension.ts

```diff
@@ -102,11 +105,33 @@ export function activate(context: ExtensionContext) {
     settings['logs']['file'] = logfile;
     return settings;
   }
+}
 
-
-
-
+function verifyConfigurations() {
+  let configXML = workspace.getConfiguration();
+  let autoCloseTags = configXML.get("xml.completion.autoCloseTags");
+  let autoClosingBrackets = configXML.get("[xml]")["editor.autoClosingBrackets"];
+  if (autoCloseTags && autoClosingBrackets != "never") {
+    window.showWarningMessage(
```

Comment:
the message briefly shows up when the extension starts, then disappears. You should probably store a copy of the config on each check. When the 1st check happens, there's no oldConfig value, you do nothing (except storing the current config into oldConfig)

As for the message, I'd rather display something like:
> "The `[xml].editor.autoClosingBrackets` setting conflicts with `xml.completion.autoCloseTags`. It's recommended to disable it."

with 2 options: `Disable` and `Don't show again`.
If disable is selected, you need to check where is autoClosingBrackets set, using configXML.inspect(key). Then update the new value at the same place (globally vs locally)
If don't show again, then you need to store a `xml.completion.autoClosingBracketsConflictWarning` globally, that you can check again on next call to verifyConfigurations()

Reply:
The message disappearing is because of didChangeConfiguration being called twice since we have intializationOptions being called (I dont think its needed anyways).

And Ill fix to match what you said for the message.